readthedocs/restapi/views/model_views.py

```diff
@@ -32,6 +33,27 @@
 log = logging.getLogger(__name__)
 
 
+class PlainTextRenderer(BaseRenderer):
+
+    """
+    Custom renderer for text/plain format.
+
+    charset is 'utf-8' by default.
+    """
+
+    media_type = 'text/plain'
+    format = 'txt'
+
+    def render(self, data, accepted_media_type=None, renderer_context=None):
+        response = renderer_context.get('response')
+        if response.status_code != status.HTTP_200_OK:
+            return data['detail'].encode(self.charset)
+        data = render_to_string(
+            'restapi/log.txt', {'build': data}
```

Comment:
@humitos for reusing the same endpoint we need to render the info from here (although we could just take the build id from the data and get the raw_log property, but I think that's unnecessary)

Some comments if we choose this path:

- the raw_log property from the build has no relevance now
- We should rename the renderer class to somthing like `PlainTextBuildRenderer`

Reply:
Also some other parts will need to be removed, but basically this is the _core_, let me know if you want me to go in this direction.

Reply:
It seems that this is the way to go. I think it's clearer to have just one endpoint for this that renders in a different format depending on the extension/format attr.

Removing the property and renaming the renderer make sense to me.